custom/icds_reports/views.py

```diff
@@ -430,6 +430,58 @@ def get_state_id_from_state_name(self, state_name):
         return SQLLocation.objects.get(name=state_name, location_type__name='state').location_id
 
 
+@location_safe
+@method_decorator([login_and_domain_required,
+                   toggles.ENABLE_ICDS_DASHBOARD_RELEASE_NOTES_UPDATE.required_decorator(),
+                   require_superuser], name='dispatch')
+class ReleaseNotesUpdateView(TemplateView):
+    page_title = 'Update Dashboard Release Notes'
+    urlname = 'update_dashboard_release_notes'
+    template_name = 'icds_reports/update_release_notes.html'
+
+    def post(self, request, *args, **kwargs):
+        data = request.FILES['dashboard_release_notes']
+        icds_file, _ = IcdsFile.objects.get_or_create(blob_id="dashboard_release_notes.pdf",
+                                                      data_type='dashboard_release_notes')
+
+        icds_file.store_file_in_blobdb(data)
+        icds_file.save()
+        messages.success(request, 'ICDS Dashboard Release Notes uploaded Successfully')
+        return redirect(self.urlname, domain=request.domain)
+
+
+@location_safe
+@method_decorator([login_and_domain_required], name='dispatch')
+class DownloadReleaseNotes(View):
+    def get(self, request, *args, **kwargs):
+        release_notes_file = IcdsFile.objects.filter(blob_id="dashboard_release_notes.pdf")\
+            .order_by('file_added').last()
+        release_notes = release_notes_file.get_file_from_blobdb()
+
+        release_date = release_notes_file.file_added.strftime('%d-%m-%Y')
+
+        response = HttpResponse(release_notes.read(), content_type='application/pdf')
+        response['Content-Disposition'] = safe_filename_header("Dashboard_release_notes_" + release_date + ".pdf")
+
+        return response
+
+
+@location_safe
+@method_decorator([login_and_domain_required], name='dispatch')
+class ReleaseDateView(View):
```

Comment:
it looks weird to create a separate view just to fetch the Date. You could use the view above with a query something like
`?type=<file/date>` and the endpoint could be generic something like `get_release_note`
if depending on the type, you return the response